src/participants/tim.md

```diff
@@ -0,0 +1,3 @@
+# Timothy Ostler
+
+Tim likes MathBio and baking.
```

Comment:
Actually could you change this to say: `Math Bio`

Reply:
I actually prefer it as it was before!

Reply:
Ok great no problem.